release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -278,6 +278,11 @@ within the {product-title} cluster for deployment. It detects changes to local
 code and deploys it to the cluster automatically, giving instant feedback to
 validate changes in real time. It supports multiple languages and frameworks.
 
+[id="ocp-4-2-CRC"]
+==== CodeReady Containers
+
+CodeReady Containers provides a local desktop instance of a minimal {product-title} 4 or newer cluster. This cluster provides developers with a minimal environment for development and testing purposes. It includes the `crc` CLI to interact with the CodeReady Containers virtual machine running the OpenShift cluster.
```

Comment:
s/provides/provide

Reply:
It's actually a singular -- "CodeReady Containers" is the name of one product/project. :)

Reply:
Yes Kevin, retained provides.